app/views/prior_authority/applications/show.html.erb

```diff
@@ -47,6 +48,23 @@
                 <% end %>
               </dd>
             </div>
+          <% elsif @application.sent_back? %>
+            <div class="govuk-summary-list__row">
+              <dt class="govuk-summary-list__key">
+                <%= t(".laa_response") %>
+              </dt>
+              <dd class="govuk-summary-list__value">
+                <% t('.sent_back_explanations_html',
+                    deadline: tag.strong(@application.resubmission_deadline.to_fs(:stamp))).each do |paragraph| %>
+                  <p><%= paragraph %></p>
+                <% end %>
+                <% if @application.further_information_explanation.present?%>
+                  <%= govuk_button_link_to(t('.update_application'),  '#TODO: CRM457-1192') %>
+                <% else %>
+                  <%= govuk_button_link_to(t('.update_application'),  edit_prior_authority_steps_check_answers_path(@application)) %>
+                <% end %>
+              </dd>
+            </div>
```

Comment:
since the show action was getting rather bloated i had broken this page out to a `/response_summaries/_sent_back.html.erb` 

[sent back partial](https://github.com/ministryofjustice/laa-submit-crime-forms/pull/662/files#diff-491e987d113885bdadb2b070e301fdac477e5b008246612153c5ebdf6c909f3fR1-R9)

["assessed" [name up for grabs] partial](https://github.com/ministryofjustice/laa-submit-crime-forms/pull/662/files#diff-ae3123d05034a4707a6684697be365f3a7a1b99ef5d7b0e1f84e641ff57f84a6R1-R22) which is [called here](https://github.com/ministryofjustice/laa-submit-crime-forms/pull/662/files#diff-0fd6692498e85ef5dd34428abbc0c28b98b80faa491251cf72e02f8a78ce7eb8R27-R30)